polaris.shopify.com/pages/api/v0/tokens/[tokens].tsx

```diff
@@ -97,67 +58,47 @@ const formatTokenGroup = (tokenGroup: TokenGroup, format: Format) => {
       .join('\n');
   }
 
-  return tokenGroup;
+  return tokenValues;
 };
 
 const handler = async (req: NextApiRequest, res: NextApiResponse) => {
   const formatParam = (req.query.format || 'json') as Format;
-  const schemeParam = (req.query.scheme || 'light') as Scheme;
+  const schemeParam = (req.query.scheme || 'light') as ColorScheme;
 
   if (typeof formatParam === 'string' && typeof schemeParam === 'string') {
-    const colorSchemeTokenGroup = colorSchemeMap[schemeParam];
-    const tokenGroupParam: any = req.query.tokens || '';
-    let tokenGroups: string[] = [];
+    const colorScheme = osColorSchemes[schemeParam] as ColorScheme;
```

Comment:
```suggestion
```
`osColorSchemes` are used for setting the associated [`color-scheme` CSS property](https://github.com/Shopify/polaris/blob/5741343151f3ec52842878d8ce607cd8d9fc0053/polaris-react/src/components/CustomProperties/styles.ts#L64). You shouldn't need that value anywhere in this file and should instead use the `schemeParam` (if I understand correctly).